lib/collection/certificate-list.js

```diff
@@ -15,7 +15,8 @@ _.inherit((
      * @param {Array} list - The list of certificate representations
      *
      * @example <caption>Create a new CertificateList</caption>
-     * var certificateList = new CertificateList({}, [
+     * var CertificateList = require('postman-collection').certificateList,
```

Comment:
should be `require('postman-collection').CertificateList`

Reply:
Fixed in #507.